src/Slot.ts

```diff
@@ -91,6 +103,27 @@ export function connectSlot<T=void, T2=void>(slotName: string, transports: Trans
         Promise.all(remoteHandlersConnected).then(() => callHandlers(data, handlers)) :
         callHandlers(data, handlers)
 
+    trigger.lazy = (
+        fistRemoteClientConnectCallback: () => void,
+        lastRemoteClientDisconnectCallback: () => void
+    ): Unsubscribe => {
+
+        onFirstRemoteClientConnect = () => fistRemoteClientConnectCallback()
```

Comment:
This doesn't seem to handle well mutiple calls to lazy or am I missing something ?

Reply:
Indeed I didn't have this in mind. I could either put a limit to one lazy at a time, or allow multiple lazy callbacks. Thanks for noticing!

Reply:
@fellowseb I fixed this in this last commit :ok_hand: